### DISCUSSION TITLE
[Weekly Review Report]: Jan  15 - 19

### DISCUSSION THREAD

**Discussion by danidi** [wikipathways/wikipathways-help#99] [curation-reports]
### Recent changes

17 pathways with changes

### Under construction

9 pathways are currently under construction, three of them were not changed in the last 30 days. 
https://classic.wikipathways.org/index.php/Pathway:WP5425: I've removed the under construction tag, it still has the needs work tag with open tasks @egonw 
https://classic.wikipathways.org/index.php/Pathway:WP5426: has not seen any changes by the main author since November, but is still tagged as under construction. It looks complete, but I'm not familiar with the topic to really assess that. Does someone know how to contact the author?
Same with https://classic.wikipathways.org/index.php/Pathway:WP5428.

### New pathways

No new pathways this week.

### Deleted Pathways

I by mistake removed the for deletion tag from https://classic.wikipathways.org/index.php/Pathway:WP5430 as I thought it would delete the pathway (was scheduled for deletion on 17th of December). I thought I had the right for deleting pathways, but cannot do so from the table here https://classic.wikipathways.org/index.php?title=Special:CurationTags&showPathwaysFor=Curation:ProposedDeletion. Not sure where to actually delete the pathways...

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by khanspers** [comment]
Thanks @danidi ! I will contact the authors for https://classic.wikipathways.org/index.php/Pathway:WP5426 and https://classic.wikipathways.org/index.php/Pathway:WP5428.

I have confirmed that you indeed have rights to delete pathways. The easiest way is to go to click the **delete** tab at the top of the actual pathway page. Im not sure deleting directly from the table works.

**Reply by khanspers** [reply]
After confirmation from the original authors, I approved WP5426 and WP5428.

**Reply by danidi** [reply]
Thank you! I've now deleted the two duplicate pathways.
